apps/web/src/environment.ts

```diff
@@ -64,4 +64,19 @@ export const Environment = {
     )
     return Currencies[code as keyof typeof Currencies]
   },
+
+  get isWireEnabled() {
+    const isEnabled = this.config<string>(
+      'NEXT_PUBLIC_WIRE_PAYMENT_ENABLED',
+      'false'
+    )
+    switch (isEnabled) {
+      case 'true':
+        return true
+      case 'false':
+        return false
+      default:
+        return isEnabled
+    }
+  },
```

Comment:
I added this switch because I was not able to get a boolean to return for the env variable, it was always a string.

Reply:
Right, I guess the `this.config` method is a bit misleading. Can remove the generic typing from it and have it always return a string, and then in cases like this where we need some other type handle it accordingly. The Firebase configs and the currency one are two examples of need something other than a string.

Reply:
`JSON.parse` seems to always return the right value as well. Was thinking of changing to: ```return typeof JSON.parse(isEnabled) === boolean ? JSON.parse(isEnabled) : false``` Do you see any issue with this?

Reply:
Just committed your suggestion, which is nice and clean. Thanks!

Reply:
Potential issue with using `JSON.parse` is that if it's an empty string it will throw an error.